src/frontend/Komponenter/Behandling/Fanemeny/Fane.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+import { Normaltekst } from 'nav-frontend-typografi';
+import navFarger from 'nav-frontend-core';
+import { ISide } from './sider';
+import { useApp } from '../../../App/context/AppContext';
+
+const StyledNavLink = styled(NavLink)`
+    border-bottom: 5px solid white;
+    color: inherit;
+    text-align: center;
+    text-decoration: none;
+    width: 100%;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+
+    :hover {
+        border-bottom: 5px solid ${navFarger.navBlaLighten20};
+
+        .typo-normal {
+            color: ${navFarger.navBla};
+        }
+    }
+
+    &.aktiv {
+        background-color: ${navFarger.navLysGra};
+        border-bottom: 5px solid ${navFarger.navBla};
+
+        .typo-normal {
+            font-weight: bold;
+        }
+    }
+`;
+
+const StyledTekst = styled(Normaltekst)`
+    border-bottom: 5px solid white;
+    color: ${navFarger.navGra60};
+    text-align: center;
+    text-decoration: none;
+    width: 100%;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+`;
+
+interface Props {
+    side: ISide;
+    behandlingId: string;
+    index: number;
+    deaktivert: boolean;
+}
+
+const Fane: React.FC<Props> = ({ side, behandlingId, index, deaktivert }) => {
+    const { gåTilUrl } = useApp();
+
+    return (
+        <>
+            {deaktivert && (
+                <StyledTekst>
```

Comment:
Bør vi "gråe ut" for å lettere forstå hvorfor man ikke kan trykke på den?

Reply:
Hmmm, hva mener du? Fanene blir grået ut slik koden er nå :p

Reply:
Sånn ser fanene ut når de er deaktivert:
![Skjermbilde 2021-12-09 kl  16 02 48](https://user-images.githubusercontent.com/32769446/145420952-e7879eb7-66de-4730-8589-7a3350581f00.png)

Reply:
Ha! Jeg som ikke følger med. Skylder på at klokka er 16 :D

Reply:
Det kan skje den beste :,)